addons/im_livechat/__manifest__.py

```diff
@@ -189,6 +189,32 @@
             'im_livechat/static/src/scss/im_livechat_bootstrap.scss',
             'im_livechat/static/src/legacy/public_livechat.scss',
             'im_livechat/static/src/legacy/public_livechat_chatbot.scss',
+
+
+            'web/static/src/core/utils/*.scss',
```

Comment:
Isn't this rule just matching `web/static/src/core/utils/transitions.scss`? I would prefer explicit files, because every css rule will potentially conflict with rules on the embedding website, so we should include as few as possible.